index.html

```diff
@@ -369,6 +369,15 @@ <h3 class="col-xs-12">Silver Level Sponsors</h3>
 					<p><a href="http://kcura.com" target="_blank">kCura</a>: Our mission is to help our customers organize data, discover the truth, and act on it. And that doesn’t just mean building great e-discovery software for managing massive volumes of electronic evidence during litigation or investigations. It means fulfilling the vision we set for our product, Relativity: to simplify and accelerate how the world conducts e-discovery by bringing the entire process and community together in one open, flexible, connected platform.</p>
 				</div>
 			</div>
+	  </div>
+	  <div class="row sponsor-level">
+			<h3 class="col-xs-12">Bronze Level Sponsors</h3>
+	    <div class="col-xs-12 col-sm-6 col-md-4">
+				<div class="logo">
+					<a href="http://femgineer.com/" target="_blank"><img src="assets/img/logos/femgineer-logo.png" /></a>
+					<p><a href="http://femgineer.com" target="_blank">Femgineer</a> is an education company that empowers forward-thinking techies of all backgrounds and promotes inclusivity in the tech industry.</p>
+				</div>
+			</div>
 			<div class="col-md-4">
```

Comment:
Can we get the sponsor-us blurb with a `col-md-8` instead of 4, so that it fills the rest of the container?